package.json

```diff
@@ -4,7 +4,7 @@
   "description": "",
   "main": "peviitor_scraper.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "postinstall": "git clone https://github.com/VACostin/peviitor_jsscraper_sites sites && cd sites && git filter-branch --prune-empty --subdirectory-filter sites HEAD && cd ../"
```

Comment:
To expand on this script:

1. It clones the whole "peviitor_jsscraper_sites" repo into "sites" directory.
2. cd sites
3. In this new repo, we filter everything but our "sites" subdirectory. NOTE: A warning will pop which pauses our script for ~10 seconds before resuming. Just let it be.
4. cd ../